client/src/app/pages/applications/analysis-wizard/set-targets.tsx

```diff
@@ -135,20 +135,28 @@ export const SetTargets: React.FC = () => {
           {t("wizard.terms.setTargets")}
         </Title>
         <Text>{t("wizard.label.setTargets")}</Text>
-        <SimpleSelect
-          width={200}
-          variant="typeahead"
-          id="action-select"
-          toggleId="action-select-toggle"
-          toggleAriaLabel="Action select dropdown toggle"
-          aria-label={"Select provider"}
-          value={provider}
-          options={["Java", "Go"]}
-          onChange={(selection) => {
-            setProvider(selection as string);
-          }}
-        />
       </TextContent>
+      <SimpleSelectTypeahead
```

Comment:
Should the `SimpleSelectTypeahead` actually be contained within the above `TextContent` to match the location of the previous `SimpleSelect`?

Reply:
... Nope.  If the new component is inside the `TextContent`, the drop down menu renders weird:
![image](https://github.com/konveyor/tackle2-ui/assets/3985964/d2283bb2-72be-4bf9-896c-6b4f20ba20d5)


outside of the `TextContent` it is fine.

Reply:
Yep, this is related to my comment at the top of this PR